snuba/datasets/errors_replacer.py

```diff
@@ -242,6 +244,97 @@ def _build_event_tombstone_replacement(
     )
 
 
+def _build_event_merge_replacement(
```

Comment:
I would call this `_build_group_rewrite_replacement`. It means merge in the context of a group merge, but for individual events you are really just rewriting the group id. This would make the method more intuitive I think.